oot-driver: use DockerImage insted of ImageStreamTag for the driverContainer

sudomakeinstall2 opened the pull request:
This commit allows the flexibility to use an on-premise private docker registry as well as the internal openshift registry.

SchSeba commented:
Hi @sudomakeinstall2 can you please introduce the change under an environment variable?

I reason I wanted to use the imageStream is that we can add a trigger to that this mean if someone update the source container (for a new driver version for example) you will get a new build started automatically

sudomakeinstall2 commented:
@SchSeba I added environment variable USE_DOCKER_IMAGE. Setting this environment variable replaces ImageStreamTag with DockerImage. I couldn't come up with a better name for the variable, I am open to suggestions.

Also I changed the tag of the `oot-source-driver ` and removed the nested namespace. This allows us to use quay which doesn't support nested namespaces.

SchSeba commented:
/ok-to-test

SchSeba commented:
Hi @sudomakeinstall2 please add `oot-driver: ` to the commit title to make the CI happy
